File: superhero_flutter/lib/models/feedback.dart
class Feedback {
  int id;
  int customerId;
  int contractorId;
  int value;
  String comment;

  Feedback.fromJson(map) {
    id = map['id'];
    customerId = map['customerId'];
    contractorId = map['contractorId'];
    value = map['value'];
    comment = map['comment'];
  }
}

File: superhero/src/main/java/com/frontguys/superhero/models/Feedback.java
package com.frontguys.superhero.models;

public class Feedback {
    private int id;
    private Integer customerId;
    private Integer contractorId;
    private Integer value;
    private String comment;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Integer getCustomerId() {
        return customerId;
    }

    public void setCustomerId(Integer customerId) {
        this.customerId = customerId;
    }

    public Integer getContractorId() {
        return contractorId;
    }

    public void setContractorId(Integer contractorId) {
        this.contractorId = contractorId;
    }

    public Integer getValue() {
        return value;
    }

    public void setValue(Integer value) {
        this.value = value;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }
}
